gateway/pom.xml

```diff
@@ -285,6 +280,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>net.jcip</groupId>
+      <artifactId>jcip-annotations</artifactId>
+      <scope>provided</scope>
```

Comment:
:question: Can this really be provided? AFAIK these annotations have a `Runtime` retention, meaning they are supposed to be available at runtime. By marking this provided here, we may not have them. I think it doesn't matter too much because nobody includes the gateway module into their project, so it's probably fine :sweat_smile: Might be confusing though.

Reply:
I think these scopes were added when I automatically when adding the maven plugin with InteliJ on the import, I will omit them both :)